sos/plugins/openswan.py

```diff
@@ -42,6 +42,9 @@ def setup(self):
         if self.get_option("ipsec-barf"):
             self.add_cmd_output("ipsec barf")
 
-        self.add_forbidden_path("/etc/ipsec.d/key[3-4].db")
```

Comment:
Is this definitely no longer needed for any shipping versions? Happy to get rid of it if so, just want to be sure.

Reply:
I personally don't know, but I assumed `self.add_forbidden_path('/etc/ipsec.d/*.db')` would take care of it, as we really shouldn't capture _any_ `.db` file.  Does the wildcard work here for us?

Reply:
Eh.. never mind, I apparently had glob-blindness yesterday ;-)

Reply:
:+1: No worries!